hnn_core/extracellular.py

```diff
@@ -65,23 +65,30 @@ def _transfer_resistance(section, electrode_pos, sigma, method):
     Returns
     -------
     vres : list
-        The resistance at each section.
+        The transfer resistance at each segment of the section.
     """
     electrode_pos = np.array(electrode_pos)  # electrode position to Numpy
+    seg_gap = section.L / (section.nseg + 1)
+    seg_ctr = np.zeros((section.nseg, 3))
 
-    start = np.array([section.x3d(0), section.y3d(0), section.z3d(0)])
-    end = np.array([section.x3d(1), section.y3d(1), section.z3d(1)])
+    sec_start = np.array([section.x3d(0), section.y3d(0), section.z3d(0)])
+    sec_end = np.array([section.x3d(1), section.y3d(1), section.z3d(1)])
+    sec_ori = sec_end - sec_start
+
+    for ii, seg in enumerate(section):
+        seg_ctr[ii, :] = sec_start + seg.x * sec_ori
```

Comment:
i'm a bit confused by this formulation. Is `sec_ori` a unit vector?

Reply:
oh `seg.x` is between 0 and 1 ...

Reply:
Busted. It was originally, but L79 would then read `... = sec_start + seg.x * sec_ori * norm(sec_end - sec_start)` so I just removed the `norm/norm`...

Reply:
I actually like this formulation better. Also avoids the division by `sec_ori` down below. Anyhow, either works :)